src/e2e/puppeteer/helpers/longPressThought.ts

```diff
@@ -0,0 +1,14 @@
+import { ElementHandle } from 'puppeteer'
+import { TIMEOUT_LONG_PRESS_THOUGHT } from '../../../constants'
+import sleep from '../../../util/sleep'
+import waitForEditable from './waitForEditable'
+
+/** Tap and hold a thought until a long press occurs. */
+const longPressThought = async (value: string) => {
+  const thought = (await waitForEditable(value)) as ElementHandle
+  await thought.touchStart()
+  await sleep(TIMEOUT_LONG_PRESS_THOUGHT + 100)
```

Comment:
Rather than wait an additional `100ms`, please wait for a specific condition (such a Redux state change or a DOM state change). Arbitrary sleeps add up over time and slow down the test suite for everyone on the team.